### ISSUE TITLE
Pass dot-separated string of package, module and function to success / failure callbacks

### ISSUE THREAD

**Issue by erikrichardlarson** [rq/rq#1595]
Could we allow the success / failure callbacks to accept a dot-separated string in addition to building it like below? https://github.com/rq/rq/blob/76ba690aafaa0fffbab1ee8f5db7c27c5327d792/rq/job.py#L128

Happy to create a PR if it makes sense.

**Comment by selwin** [comment]
Yes, would be happy to accept a PR for this. Please include tests also :)
